gnovm/cmd/gno/mod_test.go

```diff
@@ -72,6 +72,71 @@ func TestModApp(t *testing.T) {
 			simulateExternalRepo: true,
 			errShouldContain:     "fetch: writepackage: querychain",
 		},
+
+		// test gno.mod init with no module name
+		{
+			args:                 []string{"mod", "init"},
+			testDir:              "../../tests/integ/valid1",
+			simulateExternalRepo: true,
```

Comment:
Do you really need to create an external repo for `gno mod init` cases ?

Reply:
OK I thought that `simulateExternalRepo` is used to create a git external repo for dependencies and so `gno mod download` can fetch it. Sounds like it's different.